backend/src/main/resources/db/data/e2e/R__010_bgbl-1_1964_321.sql

```diff
@@ -364,7 +364,7 @@ VALUES ('<?xml version="1.0" encoding="UTF-8"?>
 </akn:akomaNtoso>
 ');
 
-UPDATE norm_manifestation SET publish_state = 'PUBLISHED' WHERE eli_norm_expression = 'eli/bund/bgbl-1/1964/321/2017-03-16/1/deu';
+UPDATE norm_manifestation SET publish_state = 'UNPUBLISHED' WHERE eli_norm_expression = 'eli/bund/bgbl-1/1964/321/2017-03-16/1/deu';
```

Comment:
Changing an existing and possibly already applied (other devs locally) migration will lead to a failed start-up of the app because the checksum changed. For me it is fine, just communicate with the dev team after merging that they should clean their db, or at least delete that migration of the flyway schema history